troupon/account/views.py

```diff
@@ -89,3 +93,45 @@ def some_view(request):
         # add the slash at the relative path's view and finished
         referer = u'/' + u'/'.join(referer[1:])
         return referer
+
+class UserSignupRequest(View):
```

Comment:
A lot of repetition going on here. Can you see if you can implement this using `FormView`.